patroni/dcs/kubernetes.py

```diff
@@ -1115,9 +1117,12 @@ def _patch_or_create(self, name: str, annotations: Dict[str, Any],
             func = functools.partial(self._api.create_namespaced_kind)
             # skip annotations with null values
             metadata['annotations'] = {k: v for k, v in annotations.items() if v is not None}
+        if use_endpoints and name != self.leader_path:
+            # prevent endpoints without traffic from being cleaned by K8s
+            metadata['annotations'][LEADER_ELECTION_RECORD_ANNOTATION_KEY] = '{}'
```

Comment:
What does `'{}'` mean?
Should we better write something more meaningful there?
For example, `'Patroni'`?

Besides that, it looks good.

Reply:
This is just a placeholder, as the value of this annotation is not important in our case. The [typical usage](https://kubernetes.io/docs/reference/labels-annotations-taints/#control-plane-alpha-kubernetes-io-leader) of the leader key is like the following:
`control-plane.alpha.kubernetes.io/leader={"holderIdentity":"controller-0","leaseDurationSeconds":15,"acquireTime":"2023-01-19T13:12:57Z","renewTime":"2023-01-19T13:13:54Z","leaderTransitions":1}`

I chose to use '{}' to maintain this convention but indicate an empty value

Reply:
After careful consideration, I have changed my mind. If we are not using leader annotations for leader election, then there is no need to maintain the convention of using a specific data structure. An empty '{}' may cause unnecessary confusion for experienced k8s users. 

Setting the value to Patroni would be more meaningful and simple.